File: Basic Java/Assignment Day 10 - Lesson 10/TestingSystem_Assignment_10/src/main/java/org/example/entity/Department.java
package org.example.entity;

public class Department {
    private int id;
    private String name;

    // Constructor mặc định không tham số
    public Department() {
    }

    // Constructor với tham số
    public Department(int id, String name) {
        this.id = id;
        this.name = name;
    }

    // Getter cho id
    public int getId() {
        return id;
    }

    // Setter cho id
    public void setId(int id) {
        this.id = id;
    }

    // Getter cho name
    public String getName() {
        return name;
    }

    // Setter cho name
    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Department [id=" + id + ", name=" + name + "]";
    }
}

File: ReactJS/Day 1 27 11 2023/TestingSystemAssignment1/Department.ts
export default class Department{
    private id: number;
    private name: string;

	constructor(id: number, name: string) {
        this.id = id;
        this.name = name;
    }

   

    /**
     * Getter id
     * @return {number}
     */
	public getId(): number {
		return this.id;
	}

    /**
     * Setter id
     * @param {number} value
     */
	public setId(value: number) {
		this.id = value;
	}


    /**
     * Getter name
     * @return {string}
     */
	public getName(): string {
		return this.name;
	}

    /**
     * Setter name
     * @param {string} value
     */
	public setName(value: string) {
		this.name = value;
	}
    

}

export let department = new Department(1, "A");
export function printIn4(department: Department) {
    console.log(`ID: ${department.getId()}, Name: ${department.getName()}`);
}
